q2_dada2/_denoise.py

```diff
@@ -99,6 +99,13 @@ def _denoise_helper(biom_fp, track_fp, hashed_feature_ids):
     df = pd.read_csv(track_fp, sep='\t', index_col=0)
     df.index.name = 'sample-id'
     df = df.rename(index=_filepath_to_sample)
+
+    if 'merged' in df:
+        percentage_merged = round(df['merged'] / df['denoised'] * 100, 2)
```

Comment:
add a line:
```
percentage_merged = percentage_merged.fillna(0)
```

otherwise NaNs render as empty space, which is just visually jarring and may raise questions with obvious answers:
![image](https://user-images.githubusercontent.com/1907564/65447526-92b68900-ddeb-11e9-95bc-721947fe25ec.png)